action.yml

```diff
@@ -2,23 +2,31 @@ name: "Deploy to Cloudflare Workers with Wrangler"
 branding:
   icon: "upload-cloud"
   color: "orange"
-description: "Deploy your Cloudflare Workers applications and sites directly from GitHub, using Wrangler"
+description: "Deploy your Cloudflare Workers and Pages projects from GitHub, using Wrangler"
 runs:
   using: "docker"
   image: "Dockerfile"
 inputs:
   apiKey:
     description: "(Legacy) Your Cloudflare API Key"
-  apiToken:
-    description: "Your Cloudflare API Token"
+    required: false
   email:
     description: "(Legacy) Your Cloudflare Email"
-  environment:
```

Comment:
I think we need to keep `environment` and pass it to all commands by default (like `wrangler secret put`). I just tried to deploy my Workers with `wrangler publish --env production`, but all secrets were created for the default environment (without `--production` suffix)

Reply:
That sounds right. I agree